lib/openshift/secret.rb

```diff
@@ -8,32 +8,32 @@ class Secret < ProjectResource
     RESOURCE = "secrets"
 
     # see #raw_resource
-    def type(user: nil, cached: true, quiet: false)
+    def type(user: nil, cached: true, quiet: true)
       obj = raw_resource(user: user, cached: cached, quiet: quiet)
       return obj['type']
     end
 
     # @param user [BushSlicer::User] the user to run cli commands with if needed
-    def bearer_token?(user: nil, cached: true, quiet: false)
+    def bearer_token?(user: nil, cached: true, quiet: true)
       type(user: user, cached: cached, quiet: quiet).include?('service-account-token') && raw_resource.dig('data', 'token')
```

Comment:
Can we hard code it to `quiet: true` just in case the caller passes in the value as `false`?